docs/proposals/working-groups/20221115-managed-kubernetes.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Working Group for Managed Kubernetes in Cluster API
+authors:
+  - "@jackfrancis"
+reviewers:
+    - "@pydctw”"
+    - "@richardcase"
+    - "@vincepri"
+    - "@sbueringer”"
+    - "@fabriziopandini"
+creation-date: 2022-11-15
+last-updated: 2022-11-15
+status: proposed
+see-also:
+  - https://github.com/kubernetes-sigs/cluster-api/issues/7494
+  - https://github.com/kubernetes-sigs/cluster-api/issues/6988
+---
+# Managed Kubernetes in Cluster API Working Group
+
+This document briefly outlines the scope, communication media, and stakeholders for a formal Working Group dedicated to defining a Cluster API-approved Managed Kubernetes solution.
+
+## Scope
+
+The charter for this Working Group will be standardizing Managed Kubernetes solutions across the Cluster API provider ecosystem.
+
+## Communication
```

Comment:
What about adding something about ensuring continuous communication between this group and the wider community? e.g report periodically status updates at the CAPI/providers office hours.
(I just want to make sure that the work of each group will be merged/accepted smoothly in the broader community)